src/main/java/com/epam/tc/hw2/trello/asserts/TrelloAssertProvider.java

```diff
@@ -0,0 +1,76 @@
+package com.epam.tc.hw2.trello.asserts;
+
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+
+import io.restassured.filter.log.LogDetail;
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import io.restassured.response.ValidatableResponse;
+import org.apache.http.HttpStatus;
+
+public class TrelloAssertProvider {
+
+    ValidatableResponse responseToCheck;
+
+    public TrelloAssertProvider() {
+    }
+
+    public static TrelloAssertProvider assertThat() {
+        return new TrelloAssertProvider();
+    }
+
+    public TrelloAssertProvider response(Response response) {
+        responseToCheck = response.then().log().ifValidationFails(LogDetail.BODY);
+        return this;
+    }
+
+    public TrelloAssertProvider and() {
+        return this;
+    }
+
+    public TrelloAssertProvider has() {
+        return this;
+    }
+
+    public TrelloAssertProvider correctJsonFormat() {
+        responseToCheck.assertThat().contentType(ContentType.JSON);
+        return this;
+    }
+
+    public TrelloAssertProvider correctId(String id) {
```

Comment:
also it's a better style to name functions with check or validation word (or the same meaning word) 
checkCorrectId for example
for understanding what it's doing

Reply:
I use such kind of names in asserts for fluent code style for example:
request().has().correctId().and().statusCode()
I rewrote it with correct names.